aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
I just had to include these migrations as we do with general "typo" fixes. Django 1.7 considers even `verbose_name` changes a thing to put into migrations.